packages/tatum/src/nft/tests/eth.spec.ts

```diff
@@ -0,0 +1,322 @@
+      console.log('Transfer nft: ', sendErc721Token)
+      expect(sendErc721Token).not.toBeNull()
+    } catch (e) {
+      console.log('Transfer nft error: ', e.response.data)
+      expect(e).not.toBeDefined()
+    }
+  })
+  it('should mint multiple NFTs', async () => {
+    try {
+      const firstTokenId = new Date().getTime()
+      const secondTokenId = firstTokenId + 1
+      const body = {
+        chain: Currency.ETH,
+        to: ['0x811dfbff13adfbc3cf653dcc373c03616d3471c9', '0x811dfbff13adfbc3cf653dcc373c03616d3471c9'],
+        tokenId: [firstTokenId.toString(), secondTokenId.toString()],
+        url: ['https://www.seznam.cz', 'https://www.seznam.cz'],
+        fromPrivateKey: '0d6c13fe5fed644dfa02512d4bffde9453dcb48873afb0b0a4c0cebce160c279',
+        contractAddress: '0xf74ff0da670D30d020638323e3C0EA9d3bE41A4C',
+        cashbackValues: [['0.1'], ['0.1']],
+        authorAddresses: [
+          ['0xe4bdce3fee7cd2d722580b0e701531bae004b85b', '0x811dfbff13adfbc3cf653dcc373c03616d3471c9'],
+          ['0xe4bdce3fee7cd2d722580b0e701531bae004b85b', '0x811dfbff13adfbc3cf653dcc373c03616d3471c9'],
+        ],
+      }
+      const mintedTokens = await mintMultipleNFTWithUri(true, body)
+      console.log('Mint multiple nfts: ', mintedTokens)
+      expect(mintedTokens).not.toBeNull()
+    } catch (e) {
+      console.log('Mint multiple nfts error: ', e.response.data)
+      expect(e).not.toBeDefined()
+    }
+  })
+  it('should mint NFT with uri', async () => {
+    try {
+      const tokenId = new Date().getTime().toString()
+      const body = {
+        chain: Currency.ETH,
+        to: '0xe4bdce3fee7cd2d722580b0e701531bae004b85b',
+        url: 'ipfs://bafkreicq2uidgocra7z64ix5cx5zwn2sixatpn7hdvzk36hnsgwatrfuc4',
+        contractAddress: '0x2A42ae2a6346eEbC7FE2b2b7f02158634d5390dc',
+        fromPrivateKey: '0d6c13fe5fed644dfa02512d4bffde9453dcb48873afb0b0a4c0cebce160c279',
+        tokenId: tokenId,
+        authorAddresses: ['0xffb28c3c7a1b19380b7e9e5A7Bbe2afF1AA7A5Ef'],
+        cashbackValues: ['0.30'],
+      }
+      const mintedToken = await mintNFTWithUri(body, {
+        testnet: true,
+      })
+      console.log('Mint nft with uri: ', mintedToken)
+      expect(mintedToken).not.toBeNull()
+    } catch (e) {
+      console.log('Mint nft with uri error: ', e.response.data)
+      expect(e).not.toBeDefined()
+    }
+  })
+  it('should burn NFT', async () => {
+    try {
+      const tokenId = new Date().getTime().toString()
+      const body = {
+        tokenId: tokenId,
+        contractAddress: '0x4de1444b31da20D863fC2CeDDD47557CE19bd284',
+        chain: Currency.ETH,
+        fromPrivateKey: '0d6c13fe5fed644dfa02512d4bffde9453dcb48873afb0b0a4c0cebce160c279',
+        fee: {
+          gasLimit: '5000000',
+          gasPrice: '100',
+        },
+      }
+      const burnt = await burnNFT(true, body)
+      expect(burnt).not.toBeNull()
+      console.log('Burn token: ', burnt)
```

Comment:
I am thinking about all the logs in try blocks. Are they going to be ever read or used? If anything in variable, then test pass, no need to read or use the value. If nothing in result, then test fails and there is nothing to be logged.

Reply:
Yeah the result of the logs is almost always the txId which you can use to fetch more information on the transaction
I agree it's not crucial but I guess it doesn't hurt as well